redis/src/commands/mod.rs

```diff
@@ -87,11 +87,21 @@ implement_commands! {
         cmd("SET").arg(key).arg(value)
     }
 
+<<<<<<< HEAD
     /// Sets multiple keys to their values.
+    #[deprecated(since = "0.24.0", note = "set_multiple was determined to be a poor name. Please use mset instead")]
```

Comment:
I'd leave the opinionating out of here ("poor name")`, maybe something like `"Renamed to mset() to more closely match the Redis command", or something.